internal/aggregator/aggregator.go

```diff
@@ -214,35 +223,56 @@ func (a *Aggregator) Aggregate(ctx context.Context) error {
 		}
 	}
 
+	log.V(3).Info("DEBUG", "report.NodePoolsUp", report.NodePoolsUp)
```

Comment:
Why output 3 different log msgs with "DEBUG" as the msg?

Reply:
Simplified into single log statement